academy/xl-cosmjs/with-keplr.md

```diff
@@ -13,17 +13,17 @@ Build applicatiosn that interact with the Keplr browser extension.
     
 In this section, you will learn more about: 
     
-* Detecting Keplr
-* Getting chain info
-* The user interaction flow
+* Detecting Keplr.
+* Getting chain information.
+* Working with the user interaction flow.
 
 </HighlightBox>
 
-CosmJS allows you to connect with [Keplr](https://chrome.google.com/webstore/detail/keplr/dmkamcknogkgcdfhhbddcghachkejeap), the widely used browser extension, to manage your private keys. In a previous section you used the command-line and CosmJS to issue commands to the Cosmos Hub Testnet. In this tutorial, you'll be working on a browser application that interacts with the Keplr extension.
+CosmJS allows you to connect with [Keplr](https://chrome.google.com/webstore/detail/keplr/dmkamcknogkgcdfhhbddcghachkejeap), the widely used browser extension, to manage your private keys. In a previous section you used the command-line and CosmJS to issue commands to the Cosmos Hub testnet. In this tutorial, you are working on a browser application that interacts with the Keplr extension.
```

Comment:
I thought that Testnet would be the name of this specific testnet.

Reply:
If there is only one we can capitalize it but if it is one of multiple testnets, then I would go with lower case.

Reply:
Ok, I will make sure it's changed accordingly.